src/TLuaInterpreter.cpp

```diff
@@ -9865,6 +9865,14 @@ int TLuaInterpreter::appendCmdLine( lua_State * L )
     return 0;
 }
 
+int TLuaInterpreter::getCmdLine( lua_State * L )
+{
+    Host * pHost = TLuaInterpreter::luaInterpreterMap[L];
+    QString curText = pHost->mpConsole->mpCommandLine->toPlainText();
+    lua_pushstring( L, curText.toLatin1().data() );
```

Comment:
You're probably right. I will need to have a look how to do this tough, barring @SlySven giving any pointers on what to do before I have the time for the digging.